docs/docsite/rst/playbooks_intro.rst

```diff
@@ -514,8 +505,6 @@ kept separate in the counts.
 If you ever want to see detailed output from successful modules as well as unsuccessful ones,
 use the ``--verbose`` flag.  This is available in Ansible 0.5 and later.
 
-Ansible playbook output is vastly upgraded if the cowsay
-package is installed.  Try it!
```

Comment:
Heh.  I'm unsure what to do with cowsay.  Should it be entirely undocumented?  Should we change the upbeat "try it!" into more of a warning?  I have no preference so you might be better off asking jimi-c for guidance on that.